projects/igniteui-angular/src/lib/badge/badge.component.ts

```diff
@@ -100,10 +100,18 @@ export class IgxBadgeComponent {
     /**
      * This allows you to disable igx-badge class. The default it's applied.
      *```html
+<<<<<<< HEAD
+     *  @ViewChild("MyBadge", { read: IgxBadgeComponent })
+     *  public badge: IgxBadgeComponent;
+     *
+     *  //...
+     *  badge.cssClass = false;
+=======
      *@ViewChild("MyBadge", { read: IgxBadgeComponent })
      *public badge: IgxBadgeComponent;
      * //...
      *badge.cssClass = false;
```

Comment:
I think this is only supposed to return the css class: let cssClass = this.badge.cssClass